packages/@okta/i18n/src/properties/login_da.properties

```diff
@@ -202,6 +202,20 @@ model.validation.field.invalid.format.ipv4 = Denne v\u00E6rdi er ikke en gyldig
 model.validation.field.invalid.format.hostname = Denne v\u00E6rdi er ikke et gyldigt v\u00E6rtsnavn
 model.validation.field.username = Kontroll\u00E9r dit brugernavn
 
+schema.validation.field.value.must.string = V\u00E6rdien skal v\u00E6re en streng
```

Comment:
There is additional strings included from [this commit into `master`](https://github.com/okta/okta-signin-widget/commit/8926c906e987e752fffd4300ccd62ef631cfd021). These are English leaks that should be resolved ASAP.

@nikhilvenkatraman-okta - can you confirm this?

Reply:
resolved offline